### ISSUE TITLE
Post Editor Template editing mode: The 'Save' keyboard shortcut doesn't work 

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#51908]
### Description

Similar to https://github.com/WordPress/gutenberg/issues/51907

When editing a template in the Post editor, the Save keyboard shortcut doesn't do anything.

As a user, I'd be very confused, as a widely used keyboard shortcut _suddenly_ stops working, with no apparent reason.

Seems to me that when there are entities to save, the Save keyboard shortcut should at the very least open the Save panel.

### Step-by-step reproduction instructions

- Edit a post.
- In the Settings panel, click the template name. A popover opens. Click 'Edit template;
- The Post editor switches to 'template editing' mode.
- Change something in the template.
- Press Cmd/Ctrl + S to save your changes.
- Observe that nothing happens.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by youknowriad** [comment]
Just tried this, I think this issue has been solved.
